samples/apps/bookinfo/rules/mixer-rule-ratings-ratelimit.yaml

```diff
@@ -1,8 +1,37 @@
-rules:
-- aspects:
-  - kind: quotas
-    params:
-      quotas:
-      - descriptorName: RequestCount
-        maxAmount: 1
-        expiration: 1s
+apiVersion: "config.istio.io/v1alpha2"
+kind: memquota
+metadata:
+  name: handler
+  namespace: istio-config-default
+spec:
+  quotas:
+  - name: requestcount.quota.istio-config-default
+    max_amount: 5000
+    valid_duration: 1s
+    overrides:
+    - dimensions:
+        destination: ratings
+      max_amount: 1
+      valid_duration: 1s
+
+---
+apiVersion: "config.istio.io/v1alpha2"
+kind: quota
+metadata:
+  name: requestcount
+  namespace: istio-config-default
+spec:
+  dimensions:
+    source: source.labels["app"] | "unknown"
+    destination: destination.labels["apps"] | "unknown"
```

Comment:
`destination.name` also works, right?

Reply:
`target.name: istio-ingress-2398531633-tj53n`  is the pod hostname so you can't write rules based on it.

Reply:
Interesting, that sounds like a bug. "name" should "istio-ingress" here, the short name of the service.